### DISCUSSION TITLE
Docker 이미지 최적화 | 아카이브-로그

### DISCUSSION THREAD

**Discussion by giscus** [jongwooo/blog#194] [Comment]
# Docker 이미지 최적화 | 아카이브-로그

Docker 컨테이너는 애플리케이션을 개발, 테스트, 배포하는데 유용하게 사용됩니다.
하지만 Docker 이미지의 용량이 클 경우 다운로드, 저장, 배포가 느려지고 네트워크 부하가 증가할 수 있습니다.
이럴 경우에는 Docker 이미지를 최적화하여 이미지 크기를 줄이는 것이 좋습니다.
이 글에서는 Docker 이미지를 최적화하는 방법에 대해 알아보겠습니다…

https://thearchivelog.dev/article/optimize-docker-image/?fbclid=PAAaZ_QDR83RcwInVxaTcSuecIYAg1MMsrx2BZVXFsFFIzubTUT3P8pCs2uCA

**Comment by hanyeonhee** [comment]
명절 연휴에 블로그 업데이트라니 🔥🔥🔥 아주 성실해 👏👏👏
